code/modules/antagonists/heretic/knowledge/void_lore.dm

```diff
@@ -243,25 +273,47 @@
 /datum/heretic_knowledge/ultimate/void_final/proc/on_life(mob/living/source, seconds_per_tick, times_fired)
 	SIGNAL_HANDLER
 
-	for(var/mob/living/carbon/close_carbon in view(5, source))
-		if(IS_HERETIC_OR_MONSTER(close_carbon))
+	for(var/atom/thing_in_range as anything in range(10, source))
+		if(!isatom(thing_in_range)) //Better safe than sorry
```

Comment:
Better safe than sorry - there's nothing you can get BUT atoms in `range`, may as well delete this continue